recipe/meta.yaml

```diff
@@ -5,10 +5,11 @@ package:
   version: {{ version }}
 
 source:
-  fn: notebook-{{ version }}.tar.gz
+  fn: {{ version }}.tar.gz
```

Comment:
> This would collide with other names if we aren't careful (is that why you had a SHA issue in the first place?)

Probably. I usually copy and paste the name and URL from GitHub. But I guess that it is OK to add the `notebook-` then. Right?